Sources/XMLCoder/Decoder/XMLDecoder.swift

```diff
@@ -151,12 +152,12 @@ open class XMLDecoder {
                 stringKey.formIndex(before: &lastNonUnderscore)
             }
             
-            let keyRange = firstNonUnderscore...lastNonUnderscore
-            let leadingUnderscoreRange = stringKey.startIndex..<firstNonUnderscore
-            let trailingUnderscoreRange = stringKey.index(after: lastNonUnderscore)..<stringKey.endIndex
+            let keyRange = firstNonUnderscore ... lastNonUnderscore
+            let leadingUnderscoreRange = stringKey.startIndex ..< firstNonUnderscore
+            let trailingUnderscoreRange = stringKey.index(after: lastNonUnderscore) ..< stringKey.endIndex
```

Comment:
Adding spaces around range operators is an interesting style I haven't seen before, including this library. In general I'd prefer to stick to having no spaces around those, which is the same as what the Swift standard library does for example. Does SwiftFormat support that?